namespace.go

```diff
@@ -190,13 +191,17 @@ func (ns *Namespace) newClient() (*amqp.Client, error) {
 
 	if ns.useWebSocket {
 		wssHost := ns.getWSSHostURI() + "$servicebus/websocket"
-		wssConn, err := websocket.Dial(wssHost, "amqp", "http://localhost/")
+		ctx, cancelfunc := context.WithTimeout(context.Background(), 10*time.Second)
```

Comment:
Can you please flow the context from the callers instead of hard-coding this.

Reply:
@jhendrixMSFT do you mean pass the context in as part of the WithWebSocket option, or do you mean changing the method call to accept ctx?

Reply:
update the containing method `newClient()` to take a context parameter (its callers already take a context).

Reply:
@jhendrixMSFT Okay, working on it now.  I will add another option to set the connection timeout as well, I don't really care for the hard-coded 10 second either.

Reply:
I don't think you need to set any timeout, just accept whatever context the caller passes us (the caller controls the lifetime).